ci-operator/step-registry/ipi/conf/azure/oidc-creds-provision/ipi-conf-azure-oidc-creds-provision-commands.sh

```diff
@@ -146,6 +146,7 @@ ccoctl azure create-all \
   --region="${REGION}" \
   --subscription-id="${AZURE_SUBSCRIPTION_ID}" \
   --tenant-id="${AZURE_TENANT_ID}" \
+  --network-resource-group-name="${SHARED_DIR}/resourcegroup" \
```

Comment:
I think here is some problems:
1. not each workflow calling this step would have `${SHARED_DIR}/resourcegroup`
2. In `azure-provision-resourcegroup` step, `${SHARED_DIR}/resourcegroup` are created to save the resource group naming for vnet creation, while the `ccoctl` command would also create an empty resource group with the same name for cluster install.

In a word, there are several resource groups for different purpose in this install, one is for vnet stuff, one is for cluster, another is for oidc storage.

Reply:
And another thing is I think we should remove the line 131, installer should be able to create the metadata.json correctly.

Reply:
Thank you. The PR [OCPBUGS-18246: Add networkResourceGroupName parameter for Azure](https://github.com/openshift/cloud-credential-operator/pull/597) for this feature has just been merged. I will debug it first. There is currently no nightly build available.